src/test/java/de/tum/in/www1/artemis/ExamIntegrationTest.java

```diff
@@ -160,4 +176,67 @@ public void testSaveExamToDatabase() throws Exception {
         assertThat(savedStudentExam.getExercises().get(1)).isEqualTo(studentExam.getExercises().get(1));
         assertThat(savedStudentExam.getExercises().get(2)).isEqualTo(studentExam.getExercises().get(2));
     }
+
+    @Test
+    @WithMockUser(username = "student1", roles = "USER")
+    public void testAll_asStudent() throws Exception {
+        this.testAllPreAuthorize();
+    }
+
+    @Test
+    @WithMockUser(username = "tutor1", roles = "TA")
+    public void testAll_asTutor() throws Exception {
+        this.testAllPreAuthorize();
+    }
+
+    private void testAllPreAuthorize() throws Exception {
+        Exam exam = ModelFactory.generateExam(course);
+        request.post("/api/courses/" + course.getId() + "/exams", exam, HttpStatus.FORBIDDEN);
+        request.put("/api/courses/" + course.getId() + "/exams", exam, HttpStatus.FORBIDDEN);
+        request.get("/api/courses/" + course.getId() + "/exams/" + exam1.getId(), HttpStatus.FORBIDDEN, Exam.class);
```

Comment:
Shouldn't a tutor be able to view a exam to a course in order to reach the Assessment Dashboard?

Reply:
At the moment I would limit the access as much as possible and open the routes only for users / roles that really need access. As more features are developed, the developer can decide which routes to open for his purposes. It is also possible that we will separate the calls to the student exams for the management and for the students.